ha/alert/alert_monitor.py

```diff
@@ -37,17 +39,17 @@ def __init__(self):
 
         # get environment variables
         self.crm_env = self._get_env()
-        # Modules like Node, Resource, Fencing
-        self.alert_event_module = None
-        # Module event like node became member or node lost
-        self.alert_event_type = None
+        _alert_event_filter = AlertEventFilter()
```

Comment:
`_alert_event_filter`
for local variable try to use syntax like 

members
--------------------
local: alert_event_filter
class internal: self._alert_event_filter
class shared outside menber: self.alert_event_filter

Reply:
done